src/igvfd/types/sample.py

```diff
@@ -34,6 +35,26 @@ class Biosample(Sample):
     base_types = ['Biosample'] + Sample.base_types
     schema = load_schema('igvfd:schemas/biosample.json')
 
+    @calculated_property(
+        schema={
+            'title': 'Sex',
+            'type': 'string',
```

Comment:
I think we should add 'enum': [male,female,unspecified,mixed] so that the schema reflects the choices when people view it.

Reply:
added